tests/tools/tflite_run/src/tflite_run.cc

```diff
@@ -356,8 +359,8 @@ int main(const int argc, char **argv)
     std::string model_basename;
     std::string backend_name = default_backend_cand;
     {
-      model_basename = basename(args.getTFLiteFilename().c_str());
-      // should remove extension(such as .tflite)
+      std::vector<char> vpath(args.getTFLiteFilename().begin(), args.getTFLiteFilename().end() + 1);
```

Comment:
@chunseoklee It could be invalid in c++03, which does not guarantee the buffer for string is continuous and terminated with null like c++11 or later. However, it is 2020, I don't think we need to support c++03. Or we may explicitly push back `\0` after copying `begin()` to `end()`.